region/norfair/crocomire/Post Crocomire Farming Room.json

```
@@ -800,6 +801,17 @@
         "Although it is possible to do it without jumping, by starting next to the Gamet farm, it is harder and more obscure."
       ]
     },
+    {
+      "link": [5, 4],
+      "name": "Crouch Jump Down Grab",
+      "requires": [
+        "h_canCrouchJumpDownGrab",
+        "canCarefulJump"
```

```suggestion
        "canTrickyJump"
```
I think `canTrickyJump` may be more appropriate for this (putting it into Very Hard). It seems pretty tricky to me; I remember attempting this a while ago and concluding it didn't work.